formats/service_manual_topic/frontend/examples/service_manual_topic.json

```diff
@@ -10,6 +10,20 @@
   "phase": "alpha",
   "analytics_identifier": null,
   "links": {
+    "parent": [
+      {
+        "content_id": "053d245d-b1cd-4807-9dbc-ce78654121d1",
+        "title": "Service manual",
+        "base_path": "/service-manual",
+        "description": "Helping government teams create and run great digital services that meet the Digital Service Standard.",
+        "api_url": "https://content-store.gov.uk/content/service-manual",
```

Comment:
Yes, good catch. It's wrong in a couple of other examples in this file… which I definitely didn't copy and paste… will fix.